### ISSUE TITLE
Performance of Layers and (don't) saving content.

### ISSUE THREAD

**Issue by creativeer** [Obox/layerswp#170]
There is something going on under the hood of your customizer, that makes this whole thing slow and choppy. I can't even save content. The widget turn grey and the content isn't saved. the 2nd thing is the customizer don't come up and the browser freezes.

That is with the newest chrome and firefox the case (i have made a screenshot of my new default screen, if i start the customizer...)

![screenshot-layers-loading-customizer](https://cloud.githubusercontent.com/assets/2089316/8030587/5bc088c6-0dc8-11e5-82d0-567d34b69be7.png)

**Comment by marcperel** [comment]
We do have some JS speed enhancements planned, and WordPress 4.3 will help a lot with JS load.

How many Layers pages do you have active on your site?

**Comment by creativeer** [comment]
well, there are roundabout 35 pages in layers now.

**Comment by marcperel** [comment]
Thanks, that's a big help :)

**Comment by creativeer** [comment]
No Problem Marc,
i was building the whole 35 pages over the weekend, after the loss on friday. So i know your layers very good :)

**Comment by marcperel** [comment]
Your problem has really got me thinking about how to backup the Sidebars and Widgets at least rudimentary so that you can get them back, even if it's a manual copy and paste in the DB. +1

**Comment by creativeer** [comment]
Im no developer, but maybe how you save the data is the bottleneck. Because what i see is, the customizer sidebar from wordpress is the first thing that comes up, very fast, but then your sidebar needs much more time to load all the stuff that is needed.

**Comment by petercostellocreative** [comment]
I'm also experiencing a problem where I can't save my page.  Everything has been working beautifully until now when I cant save any pages.

I have installed Visual Form Builder Pro and and threeWP broadcast today. 
But after deactivating them I am still unable to save.

Any ideas?
Pete

**Comment by marcperel** [comment]
Pete, 

What version are you running? And can you provide more insight into your content widget settings?

Thanks,
Marc

**Comment by petercostellocreative** [comment]
Hi Marc,
I'm running layers 1.3 on Wordpress 4.2.2. It's a multisite install.  This hasn't been a problem until today though.

Widget set up is:
Mobile Sidebar
- Search
Right Sidebar
- Search
- Recent Posts
- Categories
- RSS
Footer1
- Text
Footer3
- Text
Footer4
- Text

I was using Visual Form Builder Pro earlier today and thought it may have been that but have since deleted the forms. Uninstalled the plug in and still happening.

The URL is:
hobieclass.hobiecat.net.au

Any help would be greatly appreciated.

**Comment by petercostellocreative** [comment]
After more research. It appears that the plugin Visual Form Builder Pro 3.1 Was the culprit.

**Comment by petercostellocreative** [comment]
So it turns out. That installing *Any* new plugin breaks the layerswp customisers saving ability. Is this a known issue? Any suggestions?

**Comment by petercostellocreative** [comment]
I hope this is my last post but it appears that increasing memory to php has solved it.
define( 'WP_MEMORY_LIMIT', '128M' );

Hopefully this won't cause other issues.

**Comment by marcperel** [comment]
Thanks for all the feedback Peter :+1:

**Comment by Inkmonkey** [comment]
I recently had the same issue. None of the columns in a content widget would save.

I solved the issue by disabling the WordPress Importer plugin. Not sure why this affected saving, but I hope it helps someone.

**Comment by Inkmonkey** [comment]
Actually, it did not solve the problem. Seems like something's wrong with the AJAX, which might only affect certain server setups? This is the first time I am having this issue, but I am also working on my client's chosen hosting service instead of my usual service.

The error is as follows:

> [Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error) (admin-ajax.php, line 0)

**Comment by marcperel** [comment]
Hey @Inkmonkey have you checked whether or not WP_DEBUG is off? Usually if it throws a warning then all Ajax kinda falls dead and fails.

**Comment by Inkmonkey** [comment]
@marcperel Thanks for the reply, wp_debug is set to false.

Interestingly, I found this issue where embedded forms can cause issues with the customizer. I disabled Contact Form 7 and now everything seems to work fine again.

https://wordpress.org/support/topic/save-changes-not-working-in-layer-slider-wp-admin-page

**Comment by marcperel** [comment]
That's very interesting @Inkmonkey thanks
